controllers/cloud.redhat.com/namespacepool_controller.go

```diff
@@ -203,19 +203,20 @@ func (r *NamespacePoolReconciler) getNamespaceQuantityDelta(pool crd.NamespacePo
 
 func (r *NamespacePoolReconciler) increaseReadyNamespacesQueue(ctx context.Context, pool crd.NamespacePool, increaseSize int) error {
 	for i := 0; i < increaseSize; i++ {
-		namespaceName, err := helpers.CreateNamespace(ctx, r.client, &pool)
+		nsName, err := helpers.CreateNamespace(ctx, r.client, &pool)
+		if err != nil {
+			r.log.Error(err, "namespace/project creation failed for [%s]", nsName)
+		}
+
+		ns, err := helpers.UpdateNamespaceResources(ctx, r.client, &pool, nsName)
 		if err == nil {
-			r.log.Info(fmt.Sprintf("successfully created namespace [%s] in [%s] pool", namespaceName, pool.Name))
+			r.log.Info(fmt.Sprintf("successfully created namespace [%s] in [%s] pool", nsName, pool.Name))
 			continue
 		}
 
-		r.log.Error(err, fmt.Sprintf("error while creating namespace [%s]", namespaceName))
-		if namespaceName != "" {
-			err := helpers.UpdateAnnotations(ctx, r.client, namespaceName, helpers.AnnotationEnvError.ToMap())
-			if err != nil {
-				r.log.Error(err, "error while updating annotations on namespace", "namespace", namespaceName)
-				return err
-			}
+		r.log.Error(err, fmt.Sprintf("error while creating namespace [%s]", nsName))
```

Comment:
Maybe make this error message more clear. We've got a couple of different "error creating namespace" type errors, and in this case this isn't true. There was no error creating a namespace, there was an error adding a resource to a namespace.